nilearn/connectome/tests/test_connectivity_matrices.py

```diff
@@ -679,7 +670,7 @@ def test_connectivity_measure_generic(
 def _assert_connectivity_tangent(connectivities, conn_measure, covs):
     """Check output value properties for tangent connectivity measure \
     that they have the expected relationship \
-    to the input covariance matrices.
+    to the input covariance matrices.ConnectivityMeasure.
```

Comment:
Is `ConnectivityMeasure` part of the sentence? May be the dot before it is forgotten?